### PR TITLE
Refactor urls & add officer information page

### PR DESCRIPTION
**Pull request by rsachdevch**
- Refactor routes
- Refactor URLs in all controllers
- Add officer information controller and view
- Refactor auth/sign-in redirect
- Refactor company-lookup redirect
- __N.B.__ request_id is a placeholder until we integrate with the API

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/web/emergencyauthcodeweb/controller/eac/CompanyInformationPageController.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 @Controller
 @NextController(ListOfDirectorsController.class)
-@RequestMapping("/request-an-authcode/company/{companyNumber}/company-information")
+@RequestMapping("/auth-code-requests/company/{company_number}/confirm")
 public class CompanyInformationPageController extends BaseController {
```
Comment (Joseph-Nash) [inline src/main/java/uk/gov/companieshouse/web/emergencyauthcodeweb/controller/eac/CompanyInformationPageController.java:20]: should we rename this controller now?

Comment (rsachdevch) [reply src/main/java/uk/gov/companieshouse/web/emergencyauthcodeweb/controller/eac/CompanyInformationPageController.java:20]: Yeah, you're right, I'll probably have to rename the Officer one as well 👍

File: src/main/resources/templates/eac/officerInformation.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+<head>
+    <title>Officer Information</title>
+</head>
+<div id="temp-home-main-content"  layout:fragment="content">
+
+    <form th:action="@{''}" method="post" class="form" id="home">
```
Comment (Joseph-Nash) [inline src/main/resources/templates/eac/officerInformation.html:12]: can you change the id from home to something more specific?

Comment (rsachdevch) [reply src/main/resources/templates/eac/officerInformation.html:12]: Will do

File: src/main/resources/templates/eac/officerInformation.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+<head>
+    <title>Officer Information</title>
+</head>
+<div id="temp-home-main-content"  layout:fragment="content">
```
Comment (Joseph-Nash) [inline src/main/resources/templates/eac/officerInformation.html:10]: and this id needs to be specific

**Pull request merged**